app/controllers/notifications_controller.rb

```diff
@@ -26,33 +31,14 @@ def fetch_notifications
   end
 
   def signed_in
-    if !user_signed_in?
-      signed_in = -1
```

Comment:
This code wasn't reachable due to the `if_not_signed_in` before action.  If the intent was for this endpoint to actually return `-1` to un-authenticated users, I can restore this code and prevent the before action from acting on this action.